checkov/common/util/docs_generator.py

```diff
@@ -127,9 +128,10 @@ def add_from_repository(registry: Union[BaseCheckRegistry, BaseGraphRegistry], c
         add_from_repository(openapi_registry, "resource", "OpenAPI")
     if any(x in framework_list for x in ("all", "secrets")):
         for check_id, check_type in CHECK_ID_TO_SECRET_TYPE.items():
-            if use_bc_ids:
-                check_id = metadata_integration.get_bc_id(check_id)
-            printable_checks_list.append((check_id, check_type, "secrets", check_type, "secrets"))
+            if check_id in filtered_policy_ids or not filtered_policy_ids:
```

Comment:
```suggestion
            if not filtered_policy_ids or check_id in filtered_policy_ids:
```
I think the other way around is better/more performant, because the empty check is much faster done and will probably be the most common one for BC users.

Reply:
Good call 👍🏽